tests/test_parametrized.py

```diff
@@ -90,6 +90,47 @@ def test_unpack_some_modules_static():
     assert sim(x_semantic).shape == torch.Size([n_pix, n_pix])
 
 
+def test_pass_params_as_kwargs():
+    C = FlatLambdaCDM(name="cosmo")
+    lens = Point(cosmology=C)
+    thx, thy = torch.tensor(0.5), torch.tensor(0.5)
+    P = lens.potential(
+        thx,
+        thy,
+        z_s=torch.tensor(1.0),
```

Comment:
Does the method work if z_s and other parameters are not tensors?

Reply:
I checked, they have to be tensors. It's only when assigning them during the class `__init__` that they can be floats that will be cast automatically. This is probably for performance reasons. Don't want to be casting to tensor on every call.